### DISCUSSION TITLE
Name of the main repo problem, conflicts with stable diffusion main repo

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [hlky/stable-diffusion-webui#424] [General]
I see a name problem with the main repo. The main repo for stable-diffusion-webui is called stable diffusion. But stable diffusion is also the name for stable diffusion. So when i clone both repos, then i have a name problem already. And isn't the name of this tool stable-diffusion-webui?

And from which repo shall i pull now? Main or development? Wouldn't it be better to have just one repo? And let the development happen in branches, which then gets merged from time to time?

Kind regards
Reiner

**Comment by altryne** [comment]
Good idea, rebranding I think is.. important at some point. 
We always get called the hlky repo, it's time we have a name and a brand. 

however to address your specific point, you can specify the folder you clone into very easily during cloning so no naming issues would happen

`git clone url stable-diffusion-hlky` would work
